Source/WebCore/dom/TextDecoder.cpp

```diff
@@ -83,7 +83,7 @@ ExceptionOr<String> TextDecoder::decode(std::optional<BufferSource::VariantType>
 
 String TextDecoder::encoding() const
 {
-    return makeString(asASCIILowercase(StringView::fromLatin1(m_textEncoding.name())));
+    return StringView::fromLatin1(m_textEncoding.name()).convertToASCIILowercase();
```

Comment:
Do we really need the `::fromLatin1()` here? There is a `StringView(ASCIILiteral)` constructor that should be more efficient.

Reply:
Will fix.